dts/bindings/auxdisplay/sparkfun,serlcd.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2023 Jan Henke <Jan.Henke@taujhe.de>
+# SPDX-License-Identifier: MIT
```

Comment:
Oh, just spotted this one, I think we need explicit approval for non Apache files, could you change it to `Apache-2.0`? We have a CI check for flagging these but it was broken for a while and just got fixed. :-(

Reply:
Done